searx/static/themes/simple/svg4web.svgo.js

```diff
@@ -0,0 +1,10 @@
+// Optimize SVG for WEB usage
+
+const { extendDefaultPlugins } = require('svgo');
+module.exports = {
+  plugins: [
+    {
+      name: 'convertStyleToAttrs', // make SVG CSP-compliant
+    },
+  ]
+};
```

Comment:
With the configuration below, `searxng-wordmark.min.svg` size change from 1,45 Ko to 670 bytes.

```js
// Optimize SVG for WEB usage
module.exports = {
  plugins: [
    {
      name: 'convertStyleToAttrs', // make SVG CSP-compliant
    },
    {
      name: 'removeXMLProcInst',
    },
    {
      name: 'removeXMLNS',
    },
    {
      name: 'removeDoctype',
    },
    {
      name: 'removeUnusedNS',
    },
    {
      name: 'removeEditorsNSData',
    },
    {
      name: 'removeTitle',
    },
    {
      name: 'removeDesc',
    },
    {
      name: 'removeComments',
    },
    {
      name: 'removeMetadata',
    },
    {
      name: 'removeUnknownsAndDefaults',
    },
    {
      name: 'removeUselessStrokeAndFill',
    },
    {
      name: 'removeEmptyContainers',
    },
    {
      name: 'removeNonInheritableGroupAttrs',
    },
    {
      name: 'cleanupIDs',
    },
    {
      name: 'cleanupNumericValues',
    },
    {
      name: 'convertColors',
    },
    {
      name: 'convertPathData',
    },
    {
      name: 'convertTransform',
    },
    {
      name: 'moveElemsAttrsToGroup',
    },
    {
      name: 'moveGroupAttrsToElems',
    },
    {
      name: 'convertShapeToPath',
    },
    {
      name: 'convertEllipseToCircle',
    },
    {
      name: 'mergePaths',
    },
  ]
};
```

( `const { extendDefaultPlugins } = require('svgo');` is removed without issue )

Reply:
Thanks, I haven't read this sentence right: [The default preset of plugins is fully overridden if the `plugins` field is specified.](https://github.com/svg/svgo#configuration).

Instead of list all plugins, I will give this a try: [Use preset-default plugin to customize plugins options.](https://github.com/svg/svgo#configuration).

Reply:
fixed, see https://github.com/searxng/searxng/pull/503/commits/6bd241f1963d68af2ab8013754224ead9f77b282#diff-f1069a91f753a074d41248fff781cf543fe0b24e55fe5a4853affb21e00d598b